requirements.txt

```diff
@@ -1,3 +1,126 @@
+certifi==2020.11.8
+chardet==3.0.4
+chromedriver-py==86.0.4240.22
+colorama==0.4.4
+configparser==5.0.1
+crayons==0.4.0
+idna==2.10
+pyobjc==6.2.2
```

Comment:
A full pip freeze is not necessary here - adding a single line for pyttsx3 will suffice.